packages/fxa-settings/src/components/UnitRowTwoStepAuth/index.tsx

```diff
@@ -94,15 +94,15 @@ export const UnitRowTwoStepAuth = () => {
       headerContent={
         <ButtonIconReload
           title="Refresh two-step authentication"
-          classNames="mobileLandscape:hidden"
+          classNames="ml-2 mobileLandscape:hidden"
```

Comment:
does this look correct in rtl?

Reply:
Oh good point, gotta get in the habit of checking that. I'm going to add a checklist item to the pr template, make it part of our process.